### DISCUSSION TITLE
Please help me restore my purchase premium

### DISCUSSION THREAD

**Discussion by drtenma1990** [KaustubhPatange/Moviesy#38] [Q&A]
Number order: GPA.3384-5967-4753-75733
Date
![Uploading Screenshot_2022-12-04-17-30-13-988_com.google.android.gm.jpg…]()
: 27 nov 2022 10:51:59 a. m. GMT-4

**Comment by KaustubhPatange** [comment]
Yeah I'll be soon uploading newer version of apk which will fix this issue.

**Comment by drtenma1990** [comment]
Ok
Thanks for the quick response.

**Comment by drtenma1990** [comment]
In the meantime can you please send me a premium key?
I have PayPal or you can send me the file to upload into the app.

**Comment by KaustubhPatange** [comment]
That is the problem the server is not working for some reasons. I'll update
soon.

**Comment by drtenma1990** [comment]
What information you need so I can recover my premium?

**Comment by KaustubhPatange** [comment]
I don't need anything it is happening since my license server is offline
for past 2 days which is why you are not able to verify license.

It's not just you but plenty of people having the same problem. Give me
this week of time & I'll fix the issue.

**Comment by KaustubhPatange** [comment]
@drtenma1990 Can you try this build [Moviesy-snapshot-dd9aa97](https://github.com/KaustubhPatange/Moviesy/suites/9786551624/artifacts/470634833)? Basically this is the fix for the license server I was talking earlier. 

What you have to do is uninstall the current app & download this zip, and extract & install the apk within it & see if you premium is activated or not?

**Comment by drtenma1990** [comment]
Hi dear developer.
Which file are you referring to?
I clicked on the link and it showed me 404 page.

Thanks.

**Comment by KaustubhPatange** [comment]
You basically have to login to github.com before accessing the link in the above reply. Nevertheless here is a direct link which will expire in 5 hours https://tmpfiles.org/dl/397785/moviesy-snapshot-dd9aa97.zip

**Comment by KaustubhPatange** [comment]
@drtenma1990 Did it work?

**Comment by drtenma1990** [comment]
Dear developer yes it worked.
Thanks!

**Comment by drtenma1990** [comment]
Hi dear developer I'm having issues again since I installed a new phone.
I even tried buying the app again with gpay but it didn't work. It gives
the same error as In the past.
